plugins/polio/js/src/forms/VaccineManangementForm.tsx

```diff
@@ -0,0 +1,190 @@
+                                        }
+                                        value={round.number}
+                                    />
+                                ))}
+                            </Tabs>
+                        </Box>
+                    </Grid>
+                </Grid>
+            )}
+            <Grid container spacing={2}>
+                <Divider style={{ width: '100%' }} />
+                {/* First row: vaccine */}
+                <Grid item xs={12}>
+                    <Box mt={1} mb={1}>
+                        <Typography variant="button">
+                            {formatMessage(MESSAGES.vaccines)}
+                        </Typography>
+                    </Box>
+                </Grid>
+                <Grid
+                    container
+                    direction="row"
+                    item
+                    xs={12}
+                    spacing={2}
+                    justifyContent="center"
+                >
+                    <Grid container item xs={6}>
+                        <RoundVaccinesForm
+                            roundIndex={roundIndex}
+                            round={rounds[roundIndex]}
+                        />
+                    </Grid>
+                    <Grid item xs={3}>
+                        <Box mb={1}>
+                            <ReportingDelays accessor={accessor} />
+                        </Box>
+                    </Grid>
+                    <Grid item xs={3}>
+                        <Field
+                            label={formatMessage(MESSAGES.dateSignedVrf)}
+                            fullWidth
+                            name={`${accessor}.date_signed_vrf_received`}
+                            component={DateInput}
+                        />
+                    </Grid>
+                </Grid>
+                {/* second row: shipments */}
+                <Divider style={{ width: '100%' }} />
+                <Grid item xs={12}>
+                    <Box mt={1} mb={1}>
+                        <Typography variant="button">
+                            {formatMessage(MESSAGES.shipments)}
+                        </Typography>
+                    </Box>
+                </Grid>
+                <Grid container item xs={12}>
+                    <ShipmentsForm
+                        accessor={accessor}
+                        round={rounds[roundIndex]}
+                    />
+                </Grid>
+                {/* third row: Form A */}
+                <Divider style={{ width: '100%' }} />
+                <Grid item xs={12}>
+                    <Box mt={1} mb={1}>
+                        <Typography variant="button">
+                            {formatMessage(MESSAGES.formA)}
+                        </Typography>
+                    </Box>
+                </Grid>
+                <Grid container direction="row" item xs={12} spacing={2}>
+                    <Grid item xs={3}>
```

Comment:
Maybe use a separate component for FormA fields

Reply:
True. I'll make a second pass when updating the number input